scripts/qa_crowbarsetup.sh

```diff
@@ -79,7 +79,7 @@ function horizon_barclamp()
 
 function nova_role_prefix()
 {
-    if ! iscloudver 6M7plus ; then
+    if ! iscloudver 6plus ; then
```

Comment:
@jsuchome possible, didn't want to pollute this commit with further refactoring though..